recipes/datalabsimpleclient/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "DataLabSimpleClient" %}
```

Comment:
Why are you renaming the PyPI package? It should be `cdlclient`, no?

Reply:
That is a good question indeed. Actually this is a copy of the [DataLab recipe](https://github.com/conda-forge/datalab-feedstock/blob/main/recipe/meta.yaml). DataLab's Python package was named `cdl` because `datalab` was already used. Similarly, DataLab Simple Client package was named `cdlclient`. As the name "DataLab" was available in Conda, it was an opportunity to name properly the package (as it was done before on Debian for example).

Now, this package does not need to be as visible as DataLab itself because it is a utility package that will always be installed following instructions in DataLab's documentation. So, I'm really open to change the name back to `cdlclient` if needed.

Reply:
> DataLab's Python package was named cdl because datalab was already used.

You mean used in PyPI, right? While using the "proper" name in conda-forge is fine you should be aware that you may confuse users that are coming from PyPI to conda-forge. Ideally I would recommend to publish in conda-forge using multiple outputs and both names. Similar to what we do in https://github.com/conda-forge/importlib_metadata-feedstock/blob/main/recipe/meta.yaml

Reply:
OK, thanks for the tip (I see now that it's well documented).
I've just committed changes for creating an alias (CI is currently working on it).

Reply:
> I see now that it's well documented

That is a first. We are always behind our docs 😳

Reply:
Thanks for your review and for the merge. 👍